spec/models/product_spec.rb

```diff
@@ -245,12 +222,9 @@ def rate_field
         @product = TestProduct.create!(facility: @facility, name: "Test Product", url_name: "test")
         @price_group = FactoryGirl.create(:price_group, facility: @facility)
         @price_group2 = FactoryGirl.create(:price_group, facility: @facility)
-        @user = FactoryGirl.create(:user)
-        FactoryGirl.create(:user_price_group_member, user: @user, price_group: @price_group)
-        @user.reload
-
-        @user_price_group_ids = @user.price_groups.map(&:id)
+        @user_price_group_ids = [@price_group]
```

Comment:
I would kinda prefer not to have a variable called @user_price_group_ids since these in fact have nothing to do with users now. For future readability's sake.